docs/design/plugins.md

```diff
@@ -16,26 +16,29 @@ metadata:
   name: ab
 spec:
   resources:
-
   - name: a
     spec:
       object:
+        metadata:
+          name: ups-instance
         apiVersion: servicecatalog.k8s.io/v1beta1
         kind: ServiceInstance
-        metadata:
-          name: a
         spec:
-          foo: bar
+          clusterServiceClassExternalName: user-provided-service
+          clusterServicePlanExternalName: default
+          parameters:
+            credentials:
+              outputFromBinding: aSecret
 
   - name: a-binding
     dependsOn:
     - a
     spec:
       object:
-        apiVersion: servicecatalog.k8s.io/v1beta1
-        kind: ServiceBinding
         metadata:
           name: a-binding
+        apiVersion: servicecatalog.k8s.io/v1beta1
```

Comment:
nit: the canonical order (in all Kubernetes examples) is to have apiVersion and kind first, followed by metadata.
e.g. https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#creating-a-deployment